repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -149,6 +218,20 @@
         </xs:simpleContent>
     </xs:complexType>
 
+    <xs:complexType name="ArtifactDirectoryPatch">
```

Comment:
Probably want to remove this for now, or at least comment it out so people aren't trying to use it